process/Release.md

```diff
@@ -1,20 +1,29 @@
 **Scenario:** You want to release a new version
 
-- Add issues to release on JIRA
+- Notify team of release start
+- Set release start date to today
```

Comment:
there are several places to do this, and they are all equivalent. is there a benefit to specifying a specific one?